### DISCUSSION TITLE
Towards 1.0.0.Final

### DISCUSSION THREAD

**Discussion by gunnarmorling** [kcctl/kcctl#361] [General]
Thanks to a heroic effort by @C0urante, kcctl gained support for several new APIs in recent Kafka Connect versions, and I think with those in, we're in a good shape for finally publishing a first final release of the project, 1.0.0.Final. Specifically, I think we can pretty much take what's there right (plus perhaps the pending matrix testing PR) and release an CR (candidate release) this week. Barring any further issues, the final would be essentially a retagged version of that (plus potentially any trivial typo fixes and such).

@C0urante, is there anything in the backlog which you think really should be done before 1.0.0.Final? In terms of API/configuration surface (which we shouldn't break after the Final), I feel we're good; the one thing which bothered me was the irregularity around deleting connectors, but you've already taken care of that. Curious what you think :)

**Comment by C0urante** [comment]
I think https://github.com/kcctl/kcctl/issues/363 should be considered a blocker for 1.0.0 as it can be a little alienating right now trying to get to the bottom of how some of these commands work. Thoughts?

**Reply by gunnarmorling** [reply]
Ah yes, good call out. I think that makes sense. Added this to the 1.0.0.CR1 milestone, and will comment on the issue. Please feel free to add further issues to that milestone as you see fit. I've put in a tentative date for this release of mid of next week, but we can move this up or out, as needed.

**Reply by gunnarmorling** [reply]
With #363 being fixed, do you see any other blockers for 1.0.0CR1? If not, I'd pull the trigger at one of these days.

**Reply by C0urante** [reply]
I think that's it!

**Reply by gunnarmorling** [reply]
Awesome! Just hit the button and 1.0.0.CR1 is out. I think I'll just do a quick tweet now and then do a proper blog post for 1.0.0.Final in a week or two.
